Wrappers/Python/cil/optimisation/functions/Function.py

```diff
@@ -153,18 +166,52 @@ def centered_at(self, center):
     
     @property
     def L(self):
-        '''Lipschitz of the gradient of function f.
-        
-        L is positive real number, such that |f'(x) - f'(y)| <= L ||x-y||, assuming f: IG --> R'''
+        '''Lipschitz constant of the gradient of function f.'''
+    
```

Comment:
was the docstring wrong?

Reply:
No

Reply:
so why delete it? It may be obvious to mathematicians but not to the vast majority of people!